js/index.js

```diff
@@ -0,0 +1,36 @@
+const addToInputList = document.querySelector("#addName");
+const addButton = document.querySelector("#addNameToList");
+const addToOutputList = document.querySelector("#outputList");
+
+// Function to append 
+function addToList() {
```

Comment:
Good function name, but going forward can we use [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) except where unavoidable?

Reply:
I was able to switch "addToOutputList" from function expression to arrow function expression. I could not possibly make it more compact.

<img width="400" alt="image" src="https://user-images.githubusercontent.com/35523688/225374685-4cc54ffb-626c-4cb5-a4fe-47ecaaaa3fbb.png">

Reply:
This is a lot cleaner @damyl4sure, you can push your updated code.